src/app/learn/get-started/basics/serve-project/ios-simulator/index.html

```diff
@@ -0,0 +1,103 @@
+<stache
+  navOrder="3"
+  pageTitle="Serve to an iOS simulator">
+
+  <stache-page-summary>
+    <p>
+      To serve your SPA to an iOS simulator and debug it, follow these instructions.
+    </p>
+  </stache-page-summary>
+
+  <ol>
+    <li>
+      <p>
+        Download and install XCode. This install can take a long time.
+      </p>
+    </li>
+    <li>
+      <p>
+        Open XCode and select <strong>XCode</strong> in the toolbar.
+      </p>
+    </li>
+    <li>
+      <p>
+        Navigate to Open Developer Tool and select <strong>Simulator</strong>. This opens the iOS simulator to simulate a full iOS device.
+      </p>
+    </li>
+    <li>
+      <p>
+        Select the simulated power button to turn on the simulator.
+      </p>
+    </li>
+    <li>
+      <p>
+        Set up the simulated device to allow local development.
+      </p>
+      <ul>
+        <li>
+          <p>
+            On your iOS device, navigate to <strong>Settings</strong>, <strong>Safari</strong> and turn off <strong>Prevent Cross-Site Tracking</strong>.
```

Comment:
Should this say, "On your _simulated_ iOS device" ?

Reply:
Makes sense to me. I'll make the change.